Backend/flaskr/__init__.py

```diff
@@ -24,4 +27,62 @@ def after_request(response):
     def hello():
         return "hello world"
 
-    return app
\ No newline at end of file
+    @app.route("/users/register",methods=["POST"])
```

Comment:
change this endpoint to /register, you can use the same endpoint to register both company and user but if you prefer user then make it /users